main.go

```diff
@@ -484,8 +486,38 @@ func (carina *ClusterCommand) Get(pc *kingpin.ParseContext) (err error) {
 }
 
 // Delete a cluster
-func (carina *ClusterCommand) Delete(pc *kingpin.ParseContext) (err error) {
-	return carina.clusterApply(carina.ClusterClient.Delete)
+func (carina *CredentialsCommand) Delete(pc *kingpin.ParseContext) (err error) {
+	err = carina.clusterApply(carina.ClusterClient.Delete)
+	if err != nil {
+		fmt.Fprintln(os.Stderr, "Unable to delete cluster, not deleting credentials on disk")
+		return err
+	}
+	p, err := carina.clusterPath()
+
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Unable to locate carina config path, not deleteing credentials on disk\n")
+		return err
+	}
+
+	// TODO: Any other failsafes we should check here?
+	if p == "" || p == "/" {
+		return errors.New("Path to cluster is empty or a root path, not deleting")
+	}
+
+	_, statErr := os.Stat(p)
+	if os.IsNotExist(statErr) {
+		// Assume credentials were never on disk
+		return nil
+	}
+
+	// If the path exists but not the actual credentials, inform user
+	_, statErr = os.Stat(path.Join(p, "ca.pem"))
```

Comment:
Why the check for ca.pem? Is that a sanity check in case they've mucked with the directory's contents?